### ISSUE TITLE
ureaus

### ISSUE THREAD

**Issue by sfsheath** [p-lod/p-lod-data#5]
Uraeus preferred?

**Comment by epoehler** [comment]
Yes.Sent from my iPhoneOn Apr 20, 2024, at 10:33 AM, Sebastian Heath ***@***.***> wrote:﻿
Uraeus preferred?
